crates/wash-cli/src/common/link_cmd.rs

```diff
@@ -3,31 +3,29 @@ use std::collections::HashMap;
 use anyhow::{bail, Result};
 use serde_json::json;
 use wash_lib::cli::link::{
-    create_link, delete_link, query_links, LinkCommand, LinkDelCommand, LinkPutCommand,
-    LinkQueryCommand,
+    delete_link, get_links, put_link, LinkCommand, LinkDelCommand, LinkPutCommand, LinkQueryCommand,
 };
 use wash_lib::cli::{CommandOutput, OutputKind};
-use wash_lib::id::validate_contract_id;
-use wasmcloud_control_interface::LinkDefinition;
+use wasmcloud_control_interface::InterfaceLinkDefinition;
 
 use crate::appearance::spinner::Spinner;
 use crate::ctl::{link_del_output, links_table};
 
 /// Generate output for link put command
 pub fn link_put_output(
-    actor_id: impl AsRef<str>,
-    provider_id: impl AsRef<str>,
+    source_id: impl AsRef<str>,
+    target: impl AsRef<str>,
```

Comment:
Well, no not necessarily. So right now it's always a target ID, but in the future this might just be a group so it doesn't need to be a target.

There's some nuance there so, I just wanted to keep it `target` to be neutral for later when we can change it